dcgm/README.md

```diff
@@ -113,6 +113,8 @@ DCGM_FI_DEV_BRAND                                                 ,label
 DCGM_FI_DEV_SERIAL                                                ,label                  ,
 ```
 
+It is recommended that users configure collection of extra metrics 1000 to 1010 for GPU utilization (1001) and other information (1002 to 1010) if the default set of metrics is causing heavier resource consumption in certain set-ups.
```

Comment:
Hmm I might be lacking some insight, but reading this, I'm not really sure I understand what to do here. Are the metrics we're referring to 1001 - 1010 referring to things that is mentioned in the table above? 

Are we telling customers that incase of too much load, we disable every metric except 1001 - 1010?

Reply:
Ah, you know, I should probably use the metric names. In case of too much load we should actually only disable one metric.